### PR TITLE
Adds TIME TO START to Status tab

### PR DESCRIPTION
**Pull request by 13spacemen**
## What Does This PR Do
Adds Time To Start to Status tab

## Changelog
:cl:
tweak: Time To Start shows in the Status tab.
/:cl:

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -86,8 +86,7 @@
 
 	..()
 
-	statpanel("Lobby")
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/new_player.dm:89]: This statpane is still used by admins and many other things, including the starting delay and list of ready players (for admins).

I wouldn't be opposed to adding the time till start to status, but leave lobby as it is.

Comment (13spacemen) [reply code/modules/mob/new_player/new_player.dm:89]: The delay and list of ready players just gets transferred to the Status tab, I tested it with admin rights

Comment (moxian) [reply code/modules/mob/new_player/new_player.dm:89]: Old status tab
20220514-001038-dreamseeker

Old lobby tab
20220514-001030-dreamseeker

New status tab
20220514-001355-dreamseeker

(all screenshots taken as admin)

All the info is there.

**Comment by dearmochi** [comment]
While I appreciate the time you put in to submit PRs to Paradise, I would like you to at least consult a maintainer before submitting.

This can help the both of us save time by figuring out which PRs are viable or not.

From what I can tell, no one in leadership was consulted about the string of PRs you have authored. While there are no doubt good ones, there are also others that are an absolute no unless we're consulted.

When convenient, could you message me on Discord so we can clear this up? In the mean time, your pull requests will be put On Hold.

File: code/modules/mob/new_player/new_player.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/mob/new_player/new_player.dm:89]: This lumps all the ready players into status. That's a huge no from me.

The status tab lags some people enough to the point where we have a reduced mode. If we have a list of 100 people in it, status will become laggy to the point where the MC tab is preferable to it, and I don't say that lightly.

Comment (13spacemen) [reply code/modules/mob/new_player/new_player.dm:89]: Alright so keeping the Lobby tab for admin only stuff is that fine? @AffectedArc07

Comment (AffectedArc07) [reply code/modules/mob/new_player/new_player.dm:89]: > Alright so keeping the Lobby tab for admin only stuff is that fine? @AffectedArc07

Just add `Time till start` as a display in `Status` and leave lobby exactly how it is.

File: code/modules/mob/new_player/new_player.dm [hunk 2]
```diff
@@ -86,6 +86,11 @@
 
 	..()
 
+	if(client.statpanel=="Status" && SSticker)
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/new_player.dm:89]:
```suggestion
	if(client.statpanel == "Status" && SSticker)
```

File: code/modules/mob/new_player/new_player.dm [hunk 3]
```diff
@@ -86,6 +86,11 @@
...
+		if((SSticker.current_state == GAME_STATE_PREGAME) && SSticker.ticker_going)
+			stat("Time To Start:", round(SSticker.pregame_timeleft/10))
+		if((SSticker.current_state == GAME_STATE_PREGAME) && !SSticker.ticker_going)
+			stat("Time To Start:", "DELAYED")
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/new_player.dm:93]:
```suggestion
		if((SSticker.current_state == GAME_STATE_PREGAME) && SSticker.ticker_going)
			stat("Time To Start: [round(SSticker.pregame_timeleft/10)]")
		if((SSticker.current_state == GAME_STATE_PREGAME) && !SSticker.ticker_going)
			stat("Time To Start: DELAYED")
```

Formatting

File: code/modules/mob/new_player/new_player.dm [hunk 4]
```diff
@@ -87,6 +87,9 @@
 	..()
 
 	statpanel("Lobby")
+	if(client.statpanel == "Status" && SSticker)
+		if((SSticker.current_state == GAME_STATE_PREGAME)
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/new_player.dm:91]:
```suggestion
		if(SSticker.current_state == GAME_STATE_PREGAME)
```

Comment (13spacemen) [reply code/modules/mob/new_player/new_player.dm:91]: did it on my phone sorry bout that, fixing

File: code/modules/mob/new_player/new_player.dm [hunk 5]
```diff
@@ -87,6 +87,9 @@
 	..()
 
 	statpanel("Lobby")
+	if(client.statpanel == "Status" && SSticker)
+		if(SSticker.current_state == GAME_STATE_PREGAME)
+			stat("Time To Start: [SSticker.ticker_going ? round(SSticker.pregame_timeleft / 10) : "DELAYED"]")
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/new_player.dm:92]:
```suggestion
	if(client.statpanel == "Status" && SSticker?.current_state == GAME_STATE_PREGAME)
		stat("Time To Start: [SSticker.ticker_going ? round(SSticker.pregame_timeleft / 10) : "DELAYED"]")
```

Last tweak that can be made

**Pull request closed without merging**